test/integration/guest_user_test.rb

```diff
@@ -85,7 +85,44 @@ def setup
     end
 
     within('#price') do
-      assert page.has_content?(1200)
+      assert page.has_content?('$12.00')
+    end
+  end
+
+  test "an unauthorized user can signup" do
+    visit root_url
+    click_link_or_button('Signup')
+
+    fill_in 'signup[username]', with: 'theChosen1'
+    fill_in 'signup[password]', with: 'gryffendor'
+    fill_in 'signup[password_confirmation]', with: 'gryffendor'
+    fill_in 'signup[first_name]', with: 'Harry'
+    fill_in 'signup[last_name]', with: 'Potter'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.